docs/modules/ROOT/pages/documentation/microprofile/jwt.adoc

```diff
@@ -176,6 +176,24 @@ Then the JSON Parser search for namespace prefixed claims, remove the namespace
 ==== *custom.namespace*
 When the `custom.namespace` property is set, it will always take precedence over the default namespace and be used instead.
 
+=== Disabling Type Claim Verification
+
+_Since Payara Platform 5.2020.7_
+
+The MicroProfile JWT Authentication specification currently mandates that the type claim (`typ`) of any authorization token parsed by the container is present and is set to the `JWT` value. However, the current RFC document (https://tools.ietf.org/html/rfc7519[RFC 7519]) that defines the JWT standard states that this claim is **optional**:
+
+> 5.1.  "typ" (Type) Header Parameter
+>
+> The "typ" (type) Header Parameter defined by [JWS] and [JWE] is used by JWT applications to declare the media type [IANA.MediaTypes] of this complete JWT.  This is intended for use by the JWT application when values that are not JWTs could also be present in an application data structure that can contain a JWT object; the application can use this value to disambiguate among the different kinds of objects that might be present.
+> 
+> ...
+>
+> Use of this Header Parameter is OPTIONAL.
+
+For this reason, some third-party token issuers may generate tokens that are not compatible with the Payara Platform. The Payara Platform allows to set this verification off, so you can use the `disable.type.verification` custom property and set its value to `true` to this effect. 
```

Comment:
Shouldn't this be:

```suggestion
For this reason, some third-party token issuers may generate tokens that are not compatible with the MicroProfile JWT specification. The Payara Platform allows to set this verification off, so you can use the `disable.type.verification` custom property and set its value to `true` to this effect. 
```

Reply:
and important to mention that when propagated to 3th party services, those tokens might be rejected when they follow the MP JWT spec rules.